content/developer/reference/backend/orm/changelog.rst

```diff
@@ -4,6 +4,35 @@
 Changelog
 =========
 
+Odoo Online version 15.1
+========================
+
+- With `#79622 <https://github.com/odoo/odoo/pull/79622>`_, the :meth:`reversed` works efficiently on recordset.
+
+Odoo Online version 15.2
+========================
+
+- Remove :attr:`_sequence` attribute of :class:`BaseModel`:  Let PostgreSQL uses the default sequence of the primary key
+  `#82727 <https://github.com/odoo/odoo/pull/82727>`_
+- :meth:`~odoo.models._write` of :class:`BaseModel` doesn't raise error anymore for no existing records (`#82727 <https://github.com/odoo/odoo/pull/82727>`_)
+- Specific index types on :class:`Field`:  With `#83274 <https://github.com/odoo/odoo/pull/83274>`_ and
+  `#83015 <https://github.com/odoo/odoo/pull/83015>`_, developers can now define what type of
+  indexes can be used on fields by PostgreSQL. See the :ref:`index property <reference/fields>` of
+  `odoo.fields.Field`.
+- With `#82896 <https://github.com/odoo/odoo/pull/82896>`_, translated :class:`Field` are not prefetch by default anymore (expected model _rec_name).
```

Comment:
`(expected model _rec_name)` wdym with that part ? :thinking:

Reply:
except*. I mean that all translated field are not prefetch anymore unless it is the `_rec_name` of the model.

Reply:
okay, i get it now :)